test/validation/api/pktio/pktio.c

```diff
@@ -301,6 +301,8 @@ static void pktio_init_packet_eth_ipv4(odp_packet_t pkt, uint8_t proto)
 	ip->id = odp_cpu_to_be_16(seq);
 	ip->chksum = 0;
 	odph_ipv4_csum_update(pkt);
+	ip->frag_offset = 0;
+	ip->tos = 0;
```

Comment:
Fields should be written before checksum calculation

Reply:
Fixed in v2. No effect on the tests since the tests update the checksum again.